requirements.txt

```diff
@@ -2,10 +2,11 @@
 colour-runner==0.0.4
 coverage==4.2
 dj-database-url==0.4.1
-django>=1.8,<1.9
+django>=1.10,<1.11
```

Comment:
`requirements.txt` is usually used by project to manage requirements, whereas `setup.py` is good for libraries (and is the one used to build packages for PyPI).

As far as I can see, django is the only requirement here that's actually required during for use of this library. The others are exclusively for use with the tests, and shouldn't be added to the `setup.py`. As for adding django to it, my understanding is that it's bad form because installing this package could change the version of django that is installed, and cause confusion.